specification.md

```diff
@@ -1059,6 +1061,140 @@ Note that the referenced function definition type in this case must be `rest` (d
 
 For more information about functions, reference the [Functions definitions](#Function-Definition) section.
 
+#### Using functions for HTTP Service Invocations
+
+For a few reasons, workflow designers might see necessary making straight HTTP invocations to other systems, such as:
+
+1. Target service might not expose an OpenAPI specification descriptor;
+2. Target service might not implement RESTful interfaces;
+3. The use case requires a simple HTTP request invocation.
+
+The specification supports describing HTTP invocations via a simplified interface in the [functions definition](#Function-Definition).
+
+The HTTP invocation interface can be described in the `operation` attribute of the function definition, as shown in the example:
+
+```json
+{
+"functions": [
+  {
+    "name": "queryUserById",
+    "operation": "GET#/users/{id}",
+    "type": "http"
+  }
+]
+}
+```
+
+The `operation` attribute is composed by a string separated by the hashtag symbol (`#`):
+
+* The HTTP method used by the HTTP invocation. Can be any [valid HTTP method](https://www.rfc-editor.org/rfc/rfc9110.html#name-method-definitions) such as `GET`, `POST`, `PUT`, etc.
+* The endpoint **location path**, not the full endpoint URL. Runtimes implementations are **discouraged** to expect the full URL in this field since this is a matter of infrastructure configuration.
+
+The table below lists how to describe the HTTP parameters in the `operation` attribute:
+
+| Parameter | Description | Example |
+| --- | --- | --- |
+| Path | Path parameters are part of the location and must be enclosed with brackets (`{}`). The parameter name can be later referenced by the workflow states. | `/user/{id}` |
+| Query | [Query parameters](https://www.rfc-editor.org/rfc/rfc9110.html#section-4.2.2) must be enclosed with brackets (`{}`) after the query key separator (`=`). The parameter name can be later referenced by the workflow states. | `/user?status={status}` |
+
+Additional HTTP Headers can be passed via the `metadata` function definition. For example:
+
+```json
+{
+"functions": [
+  {
+    "name": "queryUserById",
+    "operation": "GET#/users/{id}",
+    "type": "http",
+    "metadata": {
+      "x-custom-header": "the-value"
+    }
+  }
+]
+}
+```
+
+> Runtimes are encouraged to support at least `Content-type: application/json` header when making HTTP invocations.
```

Comment:
```suggestion
> Runtimes are encouraged to support at least `Content-Type: application/json` header field for HTTP requests.
```
Also, how much freedom is allowed to runtimes here? What _MUST_ they support, what _MAY_ they support, and what _MUST NOT_ they support?